msibi/pair.py

```diff
@@ -131,6 +132,7 @@ def save_current_rdf(self, state, iteration, dr):
             The RDF bin size
         """
         rdf = self._states[state]["current_rdf"]
+        #WHAT IS THIS LINE DOING?
```

Comment:
seems like it's subtracting dr/2 from all _r_ values? I'm confused too

Reply:
Seems like this is performing the bin centering for the x values of the RDF, but I don't think we need to do this anymore now that we are using Freud for the RDFs. I removed it in the latest commit.